homeassistant/components/daikin/__init__.py

```diff
@@ -91,7 +91,7 @@ async def daikin_api_setup(hass, host):
 
     session = hass.helpers.aiohttp_client.async_get_clientsession()
     try:
-        with timeout(10):
+        with timeout(60):
```

Comment:
Maybe break out this timeout to a integration level constant since we use it in two places?